WalletWasabi/WabiSabi/Client/AmountDecomposer.cs

```diff
@@ -314,7 +318,7 @@ private static Money FindChange(IEnumerable<Output> decomposition, HashSet<Outpu
 		{
 			foreach (var (sum, count, decomp) in Decomposer.Decompose(
 				target: (long)myInputSum,
-				tolerance: MinAllowedOutputAmount + FeeRate.GetFee(ScriptType.Taproot.EstimateOutputVsize()), // Assume script type with higher cost to be more permissive.
+				tolerance: MinAllowedOutputAmount + ChangeFee,
```

Comment:
I decided to only touch this to be change fee, because at the end the sanity checks could potentially throw otherwise (which is a bugfix)

Reply:
I think the reason why this bug never came out, because it would only come out for the largest leftover selections, which we randomly discourage.